Modilfinal/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace Modilfinal
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Введите количество элементов массива");
+            int elementCount = int.Parse(Console.ReadLine());
+            int[] array = new int[elementCount];
+            Random random = new Random();
+            for (int i = 0; i < elementCount; i++)
+            {
+                array[i] = random.Next(1, 26);
+            }
+            int evencount = 0;
+            for (int i = 0; i < array.Length; i++)
+            {
+                if ((array[i] % 2) == 0)
+                {
+                    evencount = ++evencount;
```

Comment:
you can rewrite evencount = ++evencount; with just   ++evencount;  or  evencount++; this will work same